app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/analysisForm/strategyBlock/strategyBlock.scss

```diff
@@ -0,0 +1,59 @@
+.switcher-wrapper {
+  @media (max-width: $SCREEN_XS_MAX) {
+    height: 30px;
+    margin-top: 8px;
+    margin-bottom: 0;
+  }
+}
+
+.no-label {
+  @media (max-width: $SCREEN_XS_MAX) {
+    display: none;
+  }
+}
+
+.aa-strategy-option {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  box-sizing: border-box;
+  @media (max-width: $SCREEN_XS_MAX) {
+    flex-direction: column;
+  }
+}
+
+.aa-strategy-option-selector {
+  box-sizing: border-box;
+  padding-right: 0;
+  margin-right: 15px;
+  min-width: 240px;
+  width: 225px;
+
+  @media (max-width: $SCREEN_XS_MAX) {
+    width: auto;
+    margin-bottom: 0;
+  }
+}
+
+.radio-container {
+  align-items: flex-start;
+  margin-bottom: 15px;
+}
+
+.radio-description {
+  display: inline-block;
+  padding: 0 15px 0 0;
+}
+
+.form-break-line {
+  width: 1010px;
+  margin-bottom: 30px;
+  border: 1px solid $COLOR--gray-91;
+  @media (max-width: $SCREEN_MD_MAX) {
+    width: 100%;
+  }
+}
+
+.radio-children {
+  word-break: keep-all;
```

Comment:
keep-all property was designer for asian languages, what is purpose of using it there?

Reply:
Fixed